Document credential configuration

eheydrick opened the pull request:
## Pull Request Checklist

**Is this in reference to an existing issue?**
#2 and #240 

#### Purpose

Document how to securely pass AWS credentials

majormoses commented on README.md:
```diff
@@ -252,4 +252,20 @@
+AWS credentials are required to execute these checks. Starting with AWS-SDK v2 there are several
+methods of passing credentials to the check:
+
+1. Set `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables
```
I think we should include that there are multiple options here as well as an example or two as I have had to explain how to do this in the context of sensu. I mean we don't want to encourage people just exporting it the exact same way as part of the check definition...it should probably be sourced either within the check from a file or overall by the sensu process being started. They both have their advantages but sourcing on the check limits the scope of credentials needed and who gets it.

eheydrick commented:
I added an example of using a credential file and added more info on using an IAM role and restricting permissions. Thinking about it some more I think we should only mention credential file and instance profiles. Like you said it's easy to set env vars in an insecure manner. Putting them in the sensu process is also tricky compared to just dropping in a cred file.
